makiflow/layers/trainable_layers.py

```diff
@@ -850,10 +899,12 @@ def _init_train_params(self, data):
 
         self.running_mean = tf.Variable(self.running_mean.astype(np.float32), trainable=False, name=self.name_mean)
         self._named_params_dict[self.name_mean] = self.running_mean
+        self._regularize_params += [self.running_mean]
 
         self.running_variance = tf.Variable(self.running_variance.astype(np.float32), trainable=False,
                                             name=self.name_var)
         self._named_params_dict[self.name_var] = self.running_variance
+        self._regularize_params += [self.running_variance]
```

Comment:
Delete it. Running variance must not be regularized. Its value is based purely on statistics, not gradient descent.